alerting/src/test/kotlin/org/opensearch/alerting/AlertingRestTestCase.kt

```diff
@@ -899,7 +977,6 @@ abstract class AlertingRestTestCase : ODFERestTestCase() {
     fun deleteRoleAndRoleMapping(role: String, roleMapping: String) {
         deleteRoleMapping(role)
         deleteRole(role)
-        deleteRoleMapping(roleMapping)
```

Comment:
Shouldn't we be removing `deleteRoleMapping(role)` instead of `deleteRoleMapping(roleMapping)`? If not, then there is no use to take `roleMapping` as a parameter.